.github/workflows/publish-release.yml

```diff
@@ -82,15 +61,19 @@ jobs:
       - uses: actions/checkout@v3
         with:
           ref: ${{ github.sha }}
+      - name: Setup Node
```

Comment:
Huh, I didn't think Node.js was required for these publishing steps

Reply:
We use Yarn to iterate through all packages and publish them.

Reply:
Yarn comes pre-installed with the images we use. My test of the publish action completed successfully without this installed ([workflow config](https://github.com/Gudahtt/controllers/blob/main/.github/workflows/publish-release.yml#L79), [workflow run](https://github.com/Gudahtt/controllers/actions/runs/3432578378/jobs/5722016414))

Reply:
Okay, gotcha, makes sense. I've removed those here: [b7c1fbe](https://github.com/MetaMask/controllers/pull/831/commits/b7c1fbe5302a9e5ab9ec6da416dd442c49b13575)